testing/test_deep_learning_pipeline.py

```diff
@@ -99,8 +65,74 @@ def test_deep_learning_pipeline():
 
     assert last_batch_mse_train < last_batch_mse_validation
     assert last_epoch_mse_train < last_epoch_mse_validation
-    assert last_batch_mse_train < 1
-    assert last_epoch_mse_train < 1
+
+
+def test_deep_learning_pipeline_with_random_search():
+    # Given
+    data_inputs, expected_outputs = create_2d_data()
+
+    p = RandomSearch(DeepLearningPipeline(
+        Pipeline([
+            linear_model.LinearRegression()
+        ]),
```

Comment:
idem? why?

Reply:
I believe we don't need this `Pipeline` class here. 

If for any reason we are forced to use it because it wouldn' work otherwise, it seems strange and like it it would need to work anyway without a `Pipeline`.

Reply:
done in demo j